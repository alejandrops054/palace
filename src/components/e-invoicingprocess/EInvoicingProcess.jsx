import React from 'react'
import {Card, CardBody, CardTitle, CardSubtitle, CardHeader,Row, Col, CardFooter, Container, Input, Button} from 'reactstrap'
import {Select, Input as InputUI} from '../index'
import img from '../../assets/images/brand/systems-logos/blancos/logo-palaceelite.svg'

const EInvoicingProcess = () => {
    return (
        <Container>
            <Card style={{width:400}}>
                <CardHeader>
                    <CardTitle tag="h3"><p></p>E-Invoicing Process</CardTitle>
                </CardHeader>
                <CardBody>
                    <Row>
                    <Col lg="12">
                    <InputUI type="date" label="Movement Data" col={12} />
                    <Select label="Source" option="Membresia club" col={12} colIcon={0} />
                    <Select  option="Sale" col={12} colIcon={0} />
                    </Col>
                    </Row>
                    <div className="pt-3 d-flex justify-content-center">
                        <Button color="primary">Load</Button>
                        <Button color="secondary">Post</Button>
                    </div>
                </CardBody>
            </Card>
        </Container>
    )
}

export default EInvoicingProcess
