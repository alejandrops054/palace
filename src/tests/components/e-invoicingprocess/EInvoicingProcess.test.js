/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import EInvoicingProcess from '../../../components/e-invoicingprocess/EInvoicingProcess'

describe('EInvoicingProcess component', () => {
    test('debe mostrar snapshot', () => {

        const wrapper = shallow(<EInvoicingProcess />)
        expect(wrapper).toMatchSnapshot();
    })

})