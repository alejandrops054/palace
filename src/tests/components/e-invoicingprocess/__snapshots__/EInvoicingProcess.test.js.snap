// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EInvoicingProcess component debe mostrar snapshot 1`] = `
<Container
  tag="div"
>
  <Card
    style={
      Object {
        "width": 400,
      }
    }
    tag="div"
  >
    <CardHeader
      tag="div"
    >
      <CardTitle
        tag="h3"
      >
        <p />
        E-Invoicing Process
      </CardTitle>
    </CardHeader>
    <CardBody
      tag="div"
    >
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          lg="12"
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            col={12}
            label="Movement Data"
            type="date"
          />
          <Select
            col={12}
            colIcon={0}
            label="Source"
            option="Membresia club"
          />
          <Select
            col={12}
            colIcon={0}
            option="Sale"
          />
        </Col>
      </Row>
      <div
        className="pt-3 d-flex justify-content-center"
      >
        <Button
          color="primary"
          tag="button"
        >
          Load
        </Button>
        <Button
          color="secondary"
          tag="button"
        >
          Post
        </Button>
      </div>
    </CardBody>
  </Card>
</Container>
`;
